src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -110,14 +112,12 @@ public function findCollectionSet(
      */
     public function count($depth = 0, $filter = [], CollectionInterface $collection = null)
     {
-        $query = $this->getIdsQuery($depth, $filter, [], $collection, 'COUNT(DISTINCT collection.id)');
```

Comment:
Postgres demands that the `id` appears in a `groupBy` clause and fails. If, however, add the `groupBy` to the query Mysql 5.7 fails.